### DISCUSSION TITLE
Next/Image adds overflow:hidden

### DISCUSSION THREAD

**Discussion by ateufel** [vercel/next.js#20513] [Help]
I am trying to use next/image with a css drop shadow, but next automatically adds a parent div with overflow:hidden and the shadow on the image gets cut. How can i avoid this? Afaik there is no setting for disabling overflow.

**Comment by edgarasben** [comment]
I am having exactly the same issue.

**Comment by edgarasben** [comment]
Ended up adding my own parent div for styling:

```html
<div className="rounded-full shadow-md overflow-hidden h-20" >
  <Image src="/voxmedia.png" alt="Vox media" width="80px" height="80px"/>
</div>
```

**Reply by ateufel** [reply]
Yeah, I did the same, but I would rather not use a workaround with another div that could be avoided :)
